diff --git a/modules/user/user.install b/modules/user/user.install
index 4e1a3c22..32dbee22 100644
--- a/modules/user/user.install
+++ b/modules/user/user.install
@@ -573,17 +573,15 @@ function user_update_7003() {
  */
 function user_update_7005(&$sandbox) {
   $mail_field = array(
-    'type' => 'varchar',
-    'length' => 254,
+    'type' => 'blob',
+    'size' => 'normal',
     'not null' => FALSE,
-    'default' => '',
     'description' => "User's e-mail address.",
   );
   $init_field = array(
-    'type' => 'varchar',
-    'length' => 254,
+    'type' => 'blob',
+    'size' => 'normal',
     'not null' => FALSE,
-    'default' => '',
     'description' => 'E-mail address used for initial account creation.',
   );
   db_drop_index('users', 'mail');
