diff --git a/modules/user/user.install b/modules/user/user.install
index 4e1a3c22..b20b468e 100644
--- a/modules/user/user.install
+++ b/modules/user/user.install
@@ -147,10 +147,9 @@ function user_schema() {
         'description' => "User's password (hashed).",
       ),
       'mail' => array(
-        'type' => 'varchar',
-        'length' => 254,
+        'type' => 'blob',
+        'size' => 'normal',
         'not null' => FALSE,
-        'default' => '',
         'description' => "User's e-mail address.",
       ),
       'theme' => array(
@@ -218,10 +217,9 @@ function user_schema() {
         'description' => "Foreign key: {file_managed}.fid of user's picture.",
       ),
       'init' => array(
-        'type' => 'varchar',
-        'length' => 254,
+        'type' => 'blob',
+        'size' => 'normal',
         'not null' => FALSE,
-        'default' => '',
         'description' => 'E-mail address used for initial account creation.',
       ),
       'data' => array(
