diff --git a/modules/user/user.install b/modules/user/user.install
index 4e1a3c22..3c544181 100644
--- a/modules/user/user.install
+++ b/modules/user/user.install
@@ -911,3 +911,22 @@ function user_update_7018() {
 /**
  * @} End of "addtogroup updates-7.x-extra".
  */
+
+
+ /**
+ * Changes for Acra.
+ */
+function user_update_7020() {
+  db_change_field('users', 'mail', 'mail', array(
+    'type' => 'blob',
+    'size' => 'normal',
+    'not null' => FALSE,
+    'description' => "User's e-mail address.",
+  ));
+  db_change_field('users', 'init', 'init', array(
+    'type' => 'blob',
+    'size' => 'normal',
+    'not null' => FALSE,
+    'description' => "User's e-mail address.",
+  ));
+}
