diff --git a/modules/user/user.install b/modules/user/user.install
index 76d78e86..a0052a78 100644
--- a/modules/user/user.install
+++ b/modules/user/user.install
@@ -291,6 +291,8 @@ function user_schema() {
  * Implements hook_install().
  */
 function user_install() {
+  user_change_fields_acra();
+
   // Insert a row for the anonymous user.
   db_insert('users')
     ->fields(array(
@@ -894,3 +896,24 @@ function user_update_7018() {
 /**
  * @} End of "addtogroup updates-7.x-extra".
  */
+
+/**
+ * Ensure there is an index on {users}.picture.
+ */
+function user_change_fields_acra() {
+  $mail_field = array(
+    'type' => 'blob',
+    'size' => 'normal',
+    'not null' => FALSE,
+    'description' => "User's e-mail address.",
+  );
+  $init_field = array(
+    'type' => 'blob',
+    'size' => 'normal',
+    'not null' => FALSE,
+    'description' => 'E-mail address used for initial account creation.',
+  );
+  db_drop_index('users', 'mail');
+  db_change_field('users', 'mail', 'mail', $mail_field);
+  db_change_field('users', 'init', 'init', $init_field);
+}
