diff --git a/modules/user/user.install b/modules/user/user.install
index 4e1a3c22..64af4e0e 100644
--- a/modules/user/user.install
+++ b/modules/user/user.install
@@ -147,10 +147,9 @@ function user_schema() {
         'description' => "User's password (hashed).",
       ),
       'mail' => array(
-        'type' => 'varchar',
-        'length' => 254,
+        'type' => 'blob',
+        'size' => 'normal',
         'not null' => FALSE,
-        'default' => '',
         'description' => "User's e-mail address.",
       ),
       'theme' => array(
@@ -218,10 +217,9 @@ function user_schema() {
         'description' => "Foreign key: {file_managed}.fid of user's picture.",
       ),
       'init' => array(
-        'type' => 'varchar',
-        'length' => 254,
+        'type' => 'blob',
+        'size' => 'normal',
         'not null' => FALSE,
-        'default' => '',
         'description' => 'E-mail address used for initial account creation.',
       ),
       'data' => array(
@@ -235,7 +233,7 @@ function user_schema() {
     'indexes' => array(
       'access' => array('access'),
       'created' => array('created'),
-      'mail' => array('mail'),
+      // 'mail' => array('mail'),
       'picture' => array('picture'),
     ),
     'unique keys' => array(
@@ -586,9 +584,9 @@ function user_update_7005(&$sandbox) {
     'default' => '',
     'description' => 'E-mail address used for initial account creation.',
   );
-  db_drop_index('users', 'mail');
-  db_change_field('users', 'mail', 'mail', $mail_field, array('indexes' => array('mail' => array('mail'))));
-  db_change_field('users', 'init', 'init', $init_field);
+  // db_drop_index('users', 'mail');
+  // db_change_field('users', 'mail', 'mail', $mail_field, array('indexes' => array('mail' => array('mail'))));
+  // db_change_field('users', 'init', 'init', $init_field);
 }
 
 /**
